### ISSUE TITLE
generate smaller formats for printing out

### ISSUE THREAD

**Issue by lw64** [vala-lang/vala-www#208]
It would be very useful to have templates for printing out on A4 pages for example (or also this: https://infobeamer.c3voc.de/).

**Comment by lw64** [comment]
Not sure this makes sense here, though I think the styling could be reused.

**Comment by colinkiama** [comment]
Can you elaborate on this please? Do you want better print styling for the website or something else?

**Comment by lw64** [comment]
No, kind of separate from the website. What you would put on a paper to hand out to people to inform them about vala. So different content/a summary of the website.
